packages/xo-server/src/xapi/mixins/patching.js

```diff
@@ -50,6 +51,10 @@ async function _listXcpUpdates(host) {
       {}
     )
```

Comment:
What happens if this call throws?

Should we throw a proper error as well?

Reply:
I don't think we should since it's not something we "expect" to fail. So, to the final user, it should still be an "unknown error from the peer" kind of error.